egs2/kosp2e/asr1/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+./asr.sh \
+    --nj 8 \
+    --inference_nj 8 \
+    --lang ko \
+    --asr_config conf/train_asr_conformer.yaml \
+    --lm_config conf/train_lm_transformer.yaml \
+    --inference_config conf/decode_asr.yaml \
+    --stage 2 \
+    --train_set train \
+    --valid_set val \
+    --test_sets "test" \
+    --nbpe 2000 \
+    --ngpu 4 \
+    --batch_size 1 \
```

Comment:
To set batch size as 1 is a bit strange. May I if you can validate the setting?

Reply:
Also, we should not use multiple GPUs for this scale of the recipe.
Please refer to https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/run.sh